src/utils/getUniqueIdGenerator.js

```diff
@@ -0,0 +1,8 @@
+function getUniqueIdGenerator() {
+  let index = 0;
+  return () => {
+    return `mui-id-${index++}`;
```

Comment:
Oh sorry, I have overlooked the context part :+1:. I though we were going back to the previous implementation.